### PR TITLE
Login popup

### LINKED ISSUE #3: Create LoginPopup component
Problem
---
In order to allow users to login, we will need to assemble a LoginPopup component. 

Do this
---
Assemble a LoginPopup component that contains all of the appropriate fields and boxes

- The use of form and input tags are recommended.

Screen Shot 2020-08-30 at 7 28 44 PM

Create an accompanying CSS file to properly style the page

See the [Figma](https://www.figma.com/file/RAQUF8oTYuIwWnmTBHOizp/8-26-20-Version-1.0?node-id=0%3A1) for specific details as to the styling (color, size, shape, etc).

### PR DESCRIPTION
**Pull request by glkidd**
Finished the login popup component. Eventually will need to add in the API functions to actually take in the data the user enters. Resolves #3 
Screen Shot 2020-09-02 at 1 06 38 PM

### CODE REVIEW

File: app/assets/stylesheets/LoginPopup.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+.Login {
+    float: right;
+    margin-right: 1.438rem;
+    padding: 7px 8px 7px 8px;
+    border: none;
+    border-radius: 10px;
+    background: black;
+    color: white;
+    font-family: Arial, Helvetica, sans-serif;
+}
```
Comment (arafian) [inline app/assets/stylesheets/LoginPopup.css]: I think the style guidelines say to add a newline at the end of a file

File: app/javascript/components/LoginPopup.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+                </button>
+                <button className="Login">
+                    <b>Login</b>
+                </button>
+            </div>
+        )
+    }
+}
+
+export default LoginPopup;
```
Comment (arafian) [inline app/javascript/components/LoginPopup.js]: Newline

**Review by arafian — CHANGES_REQUESTED** [review]
Everything looks good to me apart from a couple newlines!

**Review by gbisbjerg — APPROVED** [review]
Just need new lines at end of file but otherwise looks good.

**Review by arafian — APPROVED** [review]

**Pull request merged**